crates/bevy_sprite/src/texture_slice/border_rect.rs

```diff
@@ -1,40 +1,41 @@
 use bevy_reflect::Reflect;
 
-/// Struct defining a [`Sprite`](crate::Sprite) border with padding values
+/// Defines the extents of the border of a rectangle.
+///
+/// This struct is used to represent thickness or offsets from the edges
+/// of a rectangle (left, right, top, and bottom), with values increasing inwards.
 #[derive(Default, Copy, Clone, PartialEq, Debug, Reflect)]
 pub struct BorderRect {
-    /// Pixel padding to the left
```

Comment:
As far as I know, this was the only hint in docs in anything related to texture slicing about the values being in pixels. So it would be great to get that information back somewhere.

Reply:
Yep that needs to be in the `TextureSlicer` docs I think. `BorderRect` is used in different coordinate spaces so it doesn't make sense to refer to pixels here.